google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -46,6 +46,7 @@ appengine.cloudsdk.java.components.missing=Selected Cloud SDK does not contain t
 appengine.cloudsdk.java.components.howtoinstall=To install, run 'gcloud components install app-engine-java'.
 appengine.cloudsdk.location.label=Cloud SDK directory\:
 appengine.cloudsdk.location.browse.window.title=Browse to Cloud SDK directory
+appengine.cloudsdk.location.badchars.message=The specified Cloud SDK directory contains invalid characters (<>:"?*) or starts or ends with spaces.
```

Comment:
I'm lost a bit in the github comment stream.  I think you said that you removed this. 

We don't know what really generated the InvalidPathException. I don't want us defining what the platform specific rules of the file system are in our strings.   You can always log the IPE.getMessage if that's helpful.